config/sets/twig/twig-underscore-to-namespace.php

```diff
@@ -163,14 +162,35 @@
         'Twig_Extensions_Node_Trans' => 'Twig\Extensions\Node\TransNode',
         'Twig_TemplateWrapper' => 'Twig\TemplateWrapper',
         'Twig_NodeInterface' => 'Twig\Node\Node',
-    ];
-
-    $rectorConfig->ruleWithConfiguration(RenameClassRector::class, $renamedClasses);
-
-    $oldClasses = array_keys($renamedClasses);
-
-    $rectorConfig->ruleWithConfiguration(
-        PseudoNamespaceToNamespaceRector::class,
-        [new PseudoNamespaceToNamespace('Twig_', $oldClasses)]
-    );
+        'Twig_Compiler' => 'Twig\\Compiler',
+        'Twig_Environment' => 'Twig\\Environment',
+        'Twig_ExpressionParser' => 'Twig\\ExpressionParser',
+        'Twig_FileExtensionEscapingStrategy' => 'Twig\\FileExtensionEscapingStrategy',
+        'Twig_Lexer' => 'Twig\\Lexer',
+        'Twig_Markup' => 'Twig\\Markup',
+        'Twig_Parser' => 'Twig\\Parser',
+        'Twig_Source' => 'Twig\\Source',
+        'Twig_Template' => 'Twig\\Template',
+        'Twig_Token' => 'Twig\\Token',
+        'Twig_TokenStream' => 'Twig\\TokenStream',
+        'Twig_Extension_GlobalsInterface' => 'Twig\\Extension\\GlobalsInterface',
+        'Twig_Extension_InitRuntimeInterface' => 'Twig\\Extension\\InitRuntimeInterface',
+        'Twig_Node_Deprecated' => 'Twig\\Node\\DeprecatedNode',
```

Comment:
Appearantly, there were few bugs in node renames, like this one :)

That's why explicit list is always better to make sure we have the values right :+1: